www-src/app/replicator/init.coffee

```diff
@@ -605,8 +605,15 @@ module.exports = class Init
 
     checkPlatformVersions: ->
         return if @passUnlessInMigration 'validPlatformVersions'
-        app.replicator.checkPlatformVersions \
-            @getCallbackTrigger 'validPlatformVersions'
+        app.replicator.checkPlatformVersions (err, response) =>
+            if err
+                if app.layout.currentView
+                    return app.layout.currentView.displayError err.message
```

Comment:
Okay, maybe a comment to explain that you expect currentView is device-name view. "displayError" is not present in all views.